change/@ni-nimble-components-67287dba-5fb3-4314-ab2a-5c68051c6ca7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It would be good to add test coverage for this. But that can be covered in #495, it's OK not to do it in this PR.